draft-ietf-quic-transport.md

```diff
@@ -2581,9 +2581,9 @@ received and processed.  A sent packet that has never been acknowledged is
 missing. The ACK frame contains any number of ACK blocks.  ACK blocks are
 ranges of acknowledged packets.
 
-Unlike TCP SACKs, QUIC acknowledgements are irrevocable.  Once a packet has
-been acknowledged, even if it does not appear in a future ACK frame,
-it remains acknowledged.
+QUIC acknowledgements are irrevocable:  Once a packet has been acknowledged,
+even if it does not appear in a future ACK frame, it remains acknowledged.
+(This is unlike TCP SACKs.)
```

Comment:
Don't include an entire sentence in a parenthetical.  It would be fine to remove the paretheses here.

Of course, now that you touched this, I notice that we don't have a TCP SACK reference here, and you can't just use an unexpanded acronym like this without both expansion and a reference.